adal/src/main/java/com/microsoft/aad/adal/DeserializationAuthenticationException.java

```diff
@@ -46,13 +39,4 @@ public DeserializationAuthenticationException(String msg) {
         super(defaultErrorCode, msg);
     }
 
-    /**
-     * Constructs a new DeserializationAuthenticationException with message and the cause exception
-     * @param details Details related to the error such as query string, request
-     *            info
-     * @param throwable {@link Throwable}
-     */
-    public DeserializationAuthenticationException(String msg, Throwable cause) {
-        super(defaultErrorCode, msg, cause);
-    }
```

Comment:
hm, this exception is introduced in serialize/deserialze, right? I'm a little bit surprised that when we're unable to deserialize that there is no case we need to pass the throwable explaining why we're unable to deserialize. 

---
In reply to: [65662413](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65662413) [](ancestors = 65662413,65640763)